contracts/Vault.vy

```diff
@@ -240,6 +240,36 @@ def approve(_spender : address, _value : uint256) -> bool:
     return True
 
 
+@external
+def increaseAllowance(_spender : address, _value : uint256) -> bool:
```

Comment:
why are you making the method signatures return bool if they always return True?

Reply:
It's what OZ uses https://github.com/OpenZeppelin/openzeppelin-contracts/blob/85334995073fff746961155113b0906b689a044c/contracts/token/ERC20/ERC20.sol#L170

Returning a value is actually cheaper for contracts because it doesn't have to perform a check that the call completed successfully, only the result of the call is True